src/app/components/vacancy-details/vacancy-details.component.ts

```diff
@@ -0,0 +1,34 @@
+import { Component, OnInit } from '@angular/core';
```

Comment:
This whole file could use some formatting. I see a lot of 2-space tabs. 
IntelliJ has the shortcut: "Ctrl-Alt-L" to auto format a file on Windows (might differ on Mac)

Reply:
That's an awesome feature. For mac it is option-command-l to auto format a file.